dags/main_summary.py

```diff
@@ -25,6 +26,18 @@
 # Running at 1am should suffice.
 dag = DAG('main_summary', default_args=default_args, schedule_interval='0 1 * * *')
 
+# We copy yesterday's main pings from telemetry_live to telemetry_stable
+# at the root of this DAG because telemetry_stable.main_v4 will become
+# the source for main_summary, etc. once we are comfortable retiring parquet
+# data imports.
+copy_deduplicate_main_ping = bigquery_etl_copy_deduplicate(
+    task_id='copy_deduplicate_main_ping',
+    target_project_id='moz-fx-data-shared-prod',
+    only_tables=['telemetry_live.main_v4'],
+    owner="jklukas@mozilla.com",
```

Comment:
this uses a mix of `'` and `"`, I think throughout this PR we should consistently use `"` unless the string contains a `"`, like python-black does.